content/en/agent/faq/how-do-i-uninstall-the-agent.md

```diff
@@ -12,7 +12,7 @@ Choose your platform to see dedicated instructions to uninstall the Agent:
 ### Debian/Ubuntu
 
 {{< tabs >}}
-{{% tab "Agent v6" %}}
+{{% tab "Agent v6 & v7" %}}
 
 ```
 sudo apt-get remove --purge datadog-agent -y
```

Comment:
This command will actually also remove the conf files of the Agent, which is unexpected (the command listed below is supposed to do that). So let's change this to:

```suggestion
sudo apt-get remove datadog-agent -y
```